frontend/packages/ux-editor/src/components/Properties/EditSubformTableColumns/ColumnElement/EditColumnElement.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState, type ReactElement } from 'react';
+import classes from './EditColumnElement.module.css';
+import { type TableColumn } from '../types/TableColumn';
+import { useTranslation } from 'react-i18next';
+import {
+  StudioActionCloseButton,
+  StudioCard,
+  StudioCombobox,
+  StudioDeleteButton,
+  StudioDivider,
+  StudioParagraph,
+  StudioTextfield,
+} from '@studio/components';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { useFormLayoutsQuery } from '../../../../hooks/queries/useFormLayoutsQuery';
+import { getAllLayoutComponents } from '../../../../utils/formLayoutUtils';
+import type { FormItem } from '../../../../types/FormItem';
+import { PadlockLockedFillIcon } from '@studio/icons';
+import { useTextResourcesQuery } from 'app-shared/hooks/queries';
+import { textResourceByLanguageAndIdSelector } from '../../../../selectors/textResourceSelectors';
+
+export type ColumnElementProps = {
+  sourceColumn: TableColumn;
+  columnNumber: number;
+  onDeleteColumn: () => void;
+  onEdit: (tableColumn: TableColumn) => void;
+  layoutSetName: string;
+};
+
+export const EditColumnElement = ({
+  sourceColumn,
+  columnNumber,
+  onDeleteColumn,
+  onEdit,
+  layoutSetName,
+}: ColumnElementProps): ReactElement => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const subFormLayout = layoutSetName;
+  const [tableColumn, setTableColumn] = useState(sourceColumn);
+  const { data: formLayouts } = useFormLayoutsQuery(org, app, subFormLayout);
+  const { data: textResources } = useTextResourcesQuery(org, app);
+
+  const textKeyValue = textResourceByLanguageAndIdSelector(
+    'nb',
+    tableColumn.headerContent,
+  )(textResources)?.value;
+  const components = formLayouts
+    ? Object.values(formLayouts).flatMap((layout) => {
+        return getAllLayoutComponents(layout);
+      })
+    : [];
+
+  const selectComponent = (values: string[]) => {
+    const selectedComponentId = values[0];
+    const selectedComponent = components.find((comp) => comp.id === selectedComponentId);
+
+    const updatedTableColumn = {
+      ...sourceColumn,
+      headerContent: selectedComponent.textResourceBindings?.title,
+      cellContent: { query: selectedComponent.dataModelBindings?.simpleBinding },
+    };
+    setTableColumn(updatedTableColumn);
+  };
+
+  return (
+    <StudioCard className={classes.wrapper}>
+      <EditColumnElementHeader columnNumber={columnNumber} />
+      <StudioCard.Content className={classes.content}>
+        <EditColumnElementComponentSelect
+          components={components}
+          onSelectComponent={selectComponent}
+        />
+
+        <StudioTextfield
+          label={
+            <>
+              <PadlockLockedFillIcon /> {t('ux_editor.modal_properties_textResourceBindings_title')}
```

Comment:
Here i would split line into two lines for more readabilty as long as we have fragment element. 😊
```
  <>
       <PadlockLockedFillIcon /> 
        t('ux_editor.modal_properties_textResourceBindings_title')}
 </>
```